### DISCUSSION TITLE
WASM demo taking forever to load

### DISCUSSION THREAD

**Discussion by ashu26jha** [unicode-org/icu4x#4630] [Q&A]
I am aware that WASM demo could take upto 40 seconds to load #2769, but unfortunately in my case it is stuck at the spinner itself for about 10 minutes now. 

I used the demo from `docs/tutorials/npm`. I did the following commands

```bash 
npm install -g webpack-dev-server
npm install -g webpack webpack-cli
npm install
npm start
```

It is stuck at  ```http://localhost:12349/```

PS: I tried following `intro_interactive.md` it works, not sure what I am missing in the demo

**Comment by sffc** [comment]
Do you see anything interesting in the web inspector?

**Reply by ashu26jha** [reply]
I was getting this error:
```bash
Failed to load resource: the server responded with a status of 404 (Not Found)
Refused to execute script from 'http://localhost:12349/dist/bundle.js' because its MIME type ('text/html') is not executable, and strict MIME type checking is enabled.
Failed to load resource: the server responded with a status of 404 (Not Found)
```

I did `npm run build` and it finally generated the `dist` folder. 

Thank you
